legacy/application/forms/EditAudioMD.php

```diff
@@ -188,6 +190,14 @@ public function startForm($p_id)
             ]);
         $this->addElement($mood);
 
+        // Add replay gain field
+        $replay_gain = new Zend_Form_Element_Hidden('replay_gain');
+        $replay_gain->class = 'input_text replay_gain_' . $p_id;
+        $replay_gain->setLabel(_('Replay Gain:'))
+            ->addDecorator('HtmlTag', ['tag' => 'div', 'style' => 'display:none'])
+            ->removeDecorator('Label');
+        $this->addElement($replay_gain);
```

Comment:
Why is it hidden? Was that controlled by the replay gain knob before it was removed? Shouldn't we remove this?